src/providers/gdal/qgsgdalprovider.cpp

```diff
@@ -1082,11 +1085,52 @@ QgsRasterIdentifyResult QgsGdalProvider::identify( const QgsPointXY &point, QgsR
       else
         results.insert( i, value );
     }
-    delete myBlock;
   }
   return QgsRasterIdentifyResult( QgsRaster::IdentifyFormatValue, results );
 }
 
+bool QgsGdalProvider::worldToPixel( double x, double y, int &col, int &row ) const
+{
```

Comment:
Can we have a small note for the reference that this solves
    Xgeo = GT(0) + Xpixel*GT(1) + Yline*GT(2)
    Ygeo = GT(3) + Xpixel*GT(4) + Yline*GT(5)
when Xgeo, Ygeo are given?